.github/workflows/_check-production-node-and-runtime.yml

```diff
@@ -0,0 +1,30 @@
+---
+# This workflow checks production version of aleph-node and aleph-runtime, ie
+# only compilation and not linking, which takes most of the time
+name: Check production node and runtime
+on:
+  workflow_call:
+
+jobs:
+  main:
+    name: Build production node and runtime
+    runs-on: [self-hosted, Linux, X64, large]
+    env:
+      RUST_BACKTRACE: full
+      RUSTC_WRAPPER: sccache
+    steps:
+      - name: Checkout aleph-node source code
+        uses: actions/checkout@v4
+
+      - name: Call action get-ref-properties
+        id: get-ref-properties
+        # yamllint disable-line rule:line-length
+        uses: Cardinal-Cryptography/github-actions/get-ref-properties@v2
+
+      - name: Install Rust toolchain
+        uses: Cardinal-Cryptography/github-actions/install-rust-toolchain@v2
+        with:
+          targets: wasm32-unknown-unknown
+
+      - name: Build production binary and runtime
```

Comment:
This should probably have a better name, since it doesn't build. Same for the comment at the beginning of this file, as it doesn't even compile. :D

Reply:
Good point, fixed